lib/xandra/cluster/control_connection.ex

```diff
@@ -312,12 +400,9 @@ defmodule Xandra.Cluster.ControlConnection do
   end
 
   # Discover the peers in the same data center as the node we're connected to.
```

Comment:
The code looks to me like we are discovering all peers from all data centres, nut just the local one. Am I missing something?

Reply:
No, you're right, outdated comment. We'll need to re-add the DC-only functionality, which we'll do through load-balancing policies.